### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by clairekrucker**
In this PR, the documentation is updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,38 +8,54 @@
...
 ```
 
-
 ## Usage
 
 ```hbs
 <Lottie
-  @path="/data.json"
+  @name='empty state'
+  @animationData={{this.args.animationData}}
```
Comment (herzzanu) [inline README.md:30]:
```suggestion
  @animationData={{this.animationData}}
```

File: README.md [hunk 2]
```diff
@@ -9,38 +8,54 @@
...
+  @path='/data.json'
+  @loop={{false}}
+  @autoplay={{false}}
+  @speed={{500}}
+  @containerId={{this.args.id}}
```
Comment (herzzanu) [inline README.md:35]:
```suggestion
  @containerId={{this.id}}
```

**Review by herzzanu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (clairekrucker) [reply README.md]: Done !

File: README.md (hunk 2 above)
Comment (clairekrucker) [reply README.md]: Done !

**Review by herzzanu — APPROVED** [review]

**Pull request merged**